localstack/config.py

```diff
@@ -418,6 +418,9 @@ def in_docker():
 LS_LOG = eval_log_type("LS_LOG")
 DEBUG = is_env_true("DEBUG") or LS_LOG in TRACE_LOG_LEVELS
 
+# allow setting custom log levels for individual loggers
+LOGGING_OVERRIDE = os.environ.get("LOGGING_OVERRIDE", "{}")
```

Comment:
Not sure about the name, maybe `LOG_LEVEL_CONFIG` or `LOGGING_CONFIG`? Since this is for now an undocumented change, it might not be that important, but those changes sometimes stay :)

Reply:
I think this name is misleading, since it sounds like it updates the _entire_ config, rather than just allowing overrides. WDYT?

Reply:
You are right, let's keep it to override for now :)